types.ts

```diff
@@ -827,15 +841,38 @@ export interface CollectionConnection {
   pageInfo: PageInfo
 }
 // Cart
+
+export interface CartDiscountCode {
+  applicable: boolean
+  code: string
+}
+export interface CartDeliveryOption {
+  code?: string
+  deliveryMethodType: keyof typeof DeliveryMethodType
+  description?: string
+  estimatedCost: MoneyV2
+  title?: string
+}
+
+export interface CartDeliveryGroup {
+  deliveryAddress: MailingAddress
+  deliveryOptions: Array<CartDeliveryOption>
```

Comment:
Most array types could be simplified from things like `Array<CartDeliveryOption>` to `CartDeliveryOption[]`